### ISSUE TITLE
Fix badges :+1:

### ISSUE THREAD

**Issue by jjasghar** [fog/fog-openstack#4]
I got the PR merged in, but we need to actually set up the badges eh eh?

Speaking of, @geemus how do we get CodeClimate and Travis and all set up? Is there a shared key or something that we use?

**Comment by Ladas** [comment]
yeah, and seems like coverage.io is also not working. Never tried to setup is myself. :-)

**Comment by jjasghar** [comment]
Seems like a great learning experience for us all :metal:

**Comment by Ladas** [comment]
:-)

**Comment by geemus** [comment]
I think maybe it is by virtue of permissions/org stuff of some sort... I'll try to figure it out presently and report back.

**Comment by geemus** [comment]
Travis one is actually good-to-go now it looks like (thanks to @seanhandley it appears). The build appears really to have an error.

**Comment by geemus** [comment]
code-climate is now going as well.

In both cases, it looks like you:
1. go to the site in question
2. sign in with github
3. go to your profile
4. (maybe) sync your repo list if it doesn't include the new repo
5. enable the repo
6. update the badge url (if needed)

**Comment by jjasghar** [comment]
Awesome thanks @geemus !

**Comment by geemus** [comment]
Tried to setup on coveralls, but the "sync repo list" part seems to hang/never return... So, not sure what's up there. Maybe somebody else will have better luck and/or whatever is making it not work will resolve if we try again later...

**Comment by seanhandley** [comment]
I'm just sorting out the build breakages for Ruby 1.9 - should be green shortly.

**Comment by geemus** [comment]
Great, thanks!

**Comment by seanhandley** [comment]
Ok, we're green on master for `1.9.3`, `2.0.0`, `2.1.0`, `2.1.1`, `2.2.0`, `2.3.0`, `jruby-1.9` and `jruby-head` :green_heart:

**Comment by geemus** [comment]
Awesome, thanks!

**Comment by Ladas** [comment]
I think I am progressing with the coveralls, maybe I will make it :-)

**Comment by Ladas** [comment]
ok so with this https://github.com/fog/fog-openstack/pull/13, coveralls should appear with next travis build.

So I guess we will see, if it will really works. :-) But it at least shows the repo under coveralls now.

**Comment by seanhandley** [comment]
I merged #13 @Ladas but not seeing anything in Coveralls...

**Comment by seanhandley** [comment]
Ah, looks like we need to do the additional travis config in #14

**Comment by geemus** [comment]
Thanks all!
